js/snackBar/content.js

```diff
@@ -1,4 +1,10 @@
-const snackBarContent = (`<div id="snackbar">Press <span id="snackbar-esc">esc</span> to exit full screen</div>`);
+const snackBarContent = (
+    `<div id="snackbar">
+       <span data-i18n-text="Press "></span>
+       <span id="snackbar-esc" data-i18n-text="esc"></span>
```

Comment:
Should this not be styled a little differently to make it more prominent? Can you talk to Noah about this?